tests/MongoDB.Driver.Core.TestHelpers/XunitExtensions/RequirePlatform.cs

```diff
@@ -0,0 +1,99 @@
+﻿/* Copyright 2020-present MongoDB Inc.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+using System;
+using System.Linq;
+using Xunit;
+
+namespace MongoDB.Driver.TestHelpers
+{
+    public enum SupportedTargetFramework
+    {
+        Net452,
+        NetCoreApp11,
+        NetCoreApp21,
+        NetCoreApp30
+    }
```

Comment:
This should be:

    public enum SupportedTargetFramework
    {
        Net452,
        NetStandard15,
        NetStandard20,
        NetStandard21
    }

`NetCoreApp11` etc... are *not* supported target frameworks. Those are frameworks we use to **test** the supported target frameworks against.

Reply:
done